utils/fasm2frames.py

```diff
@@ -142,6 +175,41 @@ def check_for_pudc_b(set_feature):
         if missing_features:
             raise fasm_assembler.FasmLookupError('\n'.join(missing_features))
 
+    if part is not None:
+        # Make a set of all used IOB tiles and sites. Look for the "STEPDOWN"
```

Comment:
Make sure there is not conflict (e.g. an IOB that requires the STEPDOWN, and an IOB that does NOT have STEPDOWN).

Reply:
Are there any IOBs that do not have STEPDOWN? How do I identify them?

Reply:
Sorry, I meant IOB's that have a IOSTANDARD that should not have STEPDOWN set, e.g. LVCMOS33.